gsa/src/web/components/structure/__tests__/footer.js

```diff
@@ -0,0 +1,35 @@
+/* Copyright (C) 2019 Greenbone Networks GmbH
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import React from 'react';
+
+import {render} from 'web/utils/testing';
+
+import Footer from '../footer';
+
+describe('Footer tests', () => {
+  test('should render footer with copyright', () => {
+    const {element} = render(<Footer />);
+
+    expect(element).toMatchSnapshot();
+    expect(element).toHaveTextContent(
+      'Greenbone Security Assistant (GSA) Copyright (C) 2009-2019 by Greenbone Networks GmbH',
```

Comment:
```suggestion
     /Greenbone Security Assistant (GSA) Copyright (C) * by Greenbone Networks GmbH/,
```

Reply:
toHaveTextContent allows to use a regexp: https://github.com/testing-library/jest-dom#tohavetextcontent

Not 100% sure if this rexp is valid and correct. IMHO We should not check for the exact years.